src/components/Home.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import Header from "./Header";
+
+import banner from "../assets/banner.jpg";
+import { homeCards } from "../constant";
+import { useNavigate } from "react-router-dom";
+
+const HomeCard = ({ name, image }) => {
+  const navigate = useNavigate();
+  return (
+    <div className="card card-shadow flex flex-col bg-white rounded-sm border p-5">
+      <h4 className="my-3">{name}</h4>
+      <img src={image} className="object-fit-cover m-auto" alt="electronics" />
+      <button
+        onClick={() => navigate("/products")}
+        className="btn-link-secondary mr-auto text mt-auto"
+      >
+        See More
+      </button>
+    </div>
+  );
+};
+
+const Home = () => {
+  return (
```

Comment:
suggestion :this seems like home page using a Home component, you can make separate files for pages for good readability